src/App.jsx

```diff
@@ -69,10 +72,13 @@ function App() {
         </section>
         <div className="addColorInputBtnContainer">
           {colors.length < 12 && (
-            <button onClick={addNewColorInput}>
+            <button className="addColorBtn" onClick={addNewColorInput}>
               <strong>+</strong> Add color
             </button>
           )}
+          <button className="resetBtn" onClick={resetTextfields}>
+          <img src="https://img.icons8.com/material-outlined/24/FFFFFF/recurring-appointment.png"/>   Reset
```

Comment:
I have installed react icons in the project. Please replace the image with '<AiOutlineReload />' the import is 'import {AiOutlineReload} from "react-icons/ai" '

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix